Automattic/wp-calypso
# Prevents upsell for Jetpack Backup for multisite installations

#### Changes proposed in this Pull Request

* Adds an additional state for multisite installations to notify users that Backup is not available.
* Removes unused file `wpcom-upsell.tsx` in the Jetpack Cloud code.
* Fixes a bug with the Scan request code that causes a fail when threats are not an array.

#### Screenshots
<img width="791" alt="Screen Shot 2020-07-15 at 3 00 34 PM" src="https://user-images.githubusercontent.com/1760168/87588306-67ba6000-c6b1-11ea-832c-8c9a6123336b.png">

#### Testing instructions



* Create a Jetpack multisite and connect it.
* With this code, visit cloud.jetpack.com and navigate to the Backup nav for the site.
* Verify Backup matches screenshot above.

**Note**: Jetpack Scan may not show the correct state for multisites. This is a known issue and is due to an API issue.

Fixes 1184358343400545-as-1184719508034178.
